packages/tools/client-debugger/client-debugger/src/FluidDebuggerLogger.ts

```diff
@@ -81,13 +149,16 @@ export class FluidDebuggerLogger extends TelemetryLogger {
 		// TODO: ability to disable the logger so this becomes a no-op
 
 		const newEvent: ITelemetryBaseEvent = this.prepareEvent(event);
-
-		postMessagesToWindow<TelemetryEventMessage>(undefined, {
+		const newMessage: TelemetryEventMessage = {
 			source: debuggerMessageSource,
 			type: "TELEMETRY_EVENT",
 			data: {
-				contents: newEvent,
+				contents: [newEvent],
 			},
-		});
+		};
+
+		// insert log into the beginning of the array to show the latest log first
+		this._telemetryLog.unshift(newEvent);
+		postMessagesToWindow<TelemetryEventMessage>(this.messageLoggingOptions, newMessage);
```

Comment:
Let's keep the first param as `undefined` for this call in particular; otherwise we get a _lot_ of console logging because telemetry events are very frequent.